.github/renovate.json5

```diff
@@ -27,5 +27,11 @@
 			matchDepTypes: ['packageManager'],
 			enabled: false,
 		},
+		{
+			description: 'Disable Node.js version updates with actions/setup-node',
+			matchDepNames: ['node'],
+			matchDepTypes: ['uses-with'],
+			enabled: false,
+		},
```

Comment:
Noting that our existing `github-actions` group uses `matchManagers: ['github-actions']` to target these specifically, but not sure that’s relevant here.